core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -71,10 +71,11 @@
                 data-testId="contentlet-tools" />
             <dot-ema-page-dropzone
                 *ngIf="
-                    es.editorData.canEditVariant &&
-                    !!es.bounds.length &&
-                    !es.editorData.device &&
-                    es.state === editorState.DRAGGING
+                    (es.editorData.canEditVariant &&
+                        !!es.bounds.length &&
+                        !es.editorData.device &&
+                        es.state === editorState.DRAGGING) ||
+                    es.state === editorState.SCROLLING
```

Comment:
This `or`  is misplaced and it looks like you don't need it

Reply:
These conditionals are just out crazy by now, really difficult to maintain or even understand.

Reply:
Pretty hard to read. Is there a way to use a computed signal with a good name?

Reply:
Maybe these conditions should be selectors in the store. There are a couple of these that are repeated

Reply:
Moved conditionals to store!